Xamarin.Essentials/SecureStorage/SecureStorage.android.cs

```diff
@@ -122,24 +122,30 @@ ISecretKey GetKey()
 
             if (!string.IsNullOrEmpty(existingKeyStr))
             {
-                var wrappedKey = Convert.FromBase64String(existingKeyStr);
+                try
+                {
+                    var wrappedKey = Convert.FromBase64String(existingKeyStr);
 
-                var unwrappedKey = UnwrapKey(wrappedKey, keyPair.Private);
-                var kp = unwrappedKey.JavaCast<ISecretKey>();
+                    var unwrappedKey = UnwrapKey(wrappedKey, keyPair.Private);
+                    var kp = unwrappedKey.JavaCast<ISecretKey>();
 
-                return kp;
+                    return kp;
+                }
+                catch (Exception ex)
```

Comment:
Is this too broad of a catch here? Should we try and identify a few security exception types to narrow this down to?

Reply:
Fixed up @Redth